### DISCUSSION TITLE
Crash after successfully installing the application

### DISCUSSION THREAD

**Discussion by jlij** [beeware/toga#1891] [Q&A] [answered]
It only happened recently, and it can be run on Windows or successfully installed on Android, but it will crash when opened. After testing, it is also the case without adding any code.

**Comment by freakboy3742** [comment]
Thanks... but there's literally nothing we can do with this report. The only thing we know about your app is that you are on windows and you have deployed to Android. You haven't told us anything about your app, you haven't told us anything *specific* about your runtime environment, and you haven't provided any logs of the crash.

When you run an app through Briefcase, you'll have access to the logs. You can also get access to those logs through `adb` directly. Those logs will contain all the details that are needed to diagnose what is going on. Unless you provide a copy of those logs, we literally can't do anything to help.

**Comment by jlij** [comment]
Sorry, this is a question of my expression. It is not on Windows, but on Android. It can be successfully deployed, but when opened, it will crash and exit. This situation only recently occurred. At first, I guessed it was due to my code, but I also used a pure framework.
[crash.log](https://github.com/beeware/toga/files/11349578/crash.log)

**Reply by freakboy3742** [reply, marked as answer]
This appress to be a manifestation of beeware/toga#1878.
